portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lina's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 id="title">Lina's Portfolio</h1>
+      <p>Welcome to my portfolio! My name is Lina, and I'm a Latina in Tech currently interning at Google as a STEP intern. I am a rising Junior at Yale!</p> 
```

Comment:
nit: Try to wrap the lines so they fit in 100 columns (you can configure whatever text editor you're using to add a 'gutter' line at that column). Super long lines are harder to edit, especially on small screens